keel-api/src/test/resources/examples/security-group-example.yml

```diff
@@ -10,13 +10,13 @@ spec:
     detail: ec2v1
   locations:
     accountName: test
+    vpcName: vpc0
     regions:
-      - us-west-2
-      - us-east-1
-  vpcName: vpc0
+    - name: us-west-2
+    - name: us-east-1
```

Comment:
This is a change to how regions are represented in security groups. But… it means they look the same as in other resources when those other resources are using default AZs. I think this is a good thing.